src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,78 @@
+@use '../../styles/boilerplate.scss'as *;
+
+.container {
+  @include prefix(user-select, none, webkit ms moz);
+  margin-bottom: 16px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+
+  /* Hide the browser's default checkbox */
+  input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+  }
+
+  /* Hover */
+  &:hover input:enabled~.checkmark {
+    background-color: $color-interaction-primary-alt-hover;
+  }
+
+  /* Checked */
+  input:checked:enabled~.checkmark {
+    background-color: $color-interaction-primary;
+  }
+
+  /* Checkbox & Hover */
+  input:checked:hover:enabled~.checkmark {
+    background-color: $color-interaction-primary-hover;
+  }
+
+  /* Show Check */
+  input:checked~.checkmark:after {
+    display: block;
+  }
+
+}
+
+.label {
+  @include font-size(16);
+  @include primary-font();
+  color: $color-text-main;
+  padding-left: 16px;
+}
+
+/* Custom Checkbox */
+.checkmark {
+  position: relative;
+  height: 22px;
+  width: 22px;
+  background-color: $color-interaction-primary-alt;
+  border: $color-interaction-primary solid 2px;
+  border-radius: 4px;
+
+  /* Disabled Checkbox */
+  &_disabled {
+    background-color: $color-interaction-primary-alt-disabled;
+    border: $color-interaction-primary-disabled solid 2px;
+  }
+
+  &:after {
+    @include prefix(transform, rotate(45deg), webkit ms moz);
```

Comment:
Pretty sure prefixes are no longer necessary on `transform`.

Reply:
i still see it on here - https://shouldiprefix.com/#transforms

do you know where you saw that? be happy to take it out if that's posted somewhere official .

Reply:
Interesting. If shouldiprefix.com has been reliable for you, feel free to continue using it. I'd usually refer to caniuse.com and mdn, which show that modern browsers (at least the ones we care about) don't need it:
https://caniuse.com/transforms2d
https://developer.mozilla.org/en-US/docs/Web/CSS/transform#browser_compatibility

Reply:
oh nice yea those are great. I'll take it out